awswrangler/catalog/_create.py

```diff
@@ -986,7 +986,7 @@ def create_json_table(  # pylint: disable=too-many-arguments
     columns_types: Dict[str, str],
     table_type: Optional[str] = None,
     partitions_types: Optional[Dict[str, str]] = None,
-    bucketing_info: Optional[Tuple[List[str], int]] = None,
+    bucketing_info: Optional[typing.BucketingInfoTuple] = None,
```

Comment:
Should we change the method documentation as well with this? If so, this applies to a few other methods across the PR

Reply:
It doesn't change the type, it only gives us an alias so that if we end up changing the type later on, we can do it in a single place. With that in mind, I think it's more helpful for the customer reading the docs if the docs show the actual type of the parameter, which in this case will remain `Tuple[List[str], int]`.

I think a bigger discussion here is whether there's a value in changing this to a `NamedTuple` or a `TypedDict` instead. I don't think having a tuple as a parameter is intuitive, especially since the two values represent different things.

To give an example, this code

```python
to_sql(
    ...,
    bucketing_info=(["col1"], 4),
)
```

is not as readable as

```python
to_sql(
    ...,
    bucketing_info={
        columns=["col1"],
        num_buckets=4,
    }
)
```

Either way, I think a change like this deserves its own PR (and its own discussion).

Reply:
Good point! I like the idea of a TypeDict for this, something to discuss in a separate PR though as you say